homeassistant/components/bring/todo.py

```diff
@@ -0,0 +1,186 @@
+"""Todo platform for the Bring! integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+from typing import TYPE_CHECKING
+
+from python_bring_api.exceptions import BringRequestException
+from python_bring_api.types import BringList
+
+from homeassistant import config_entries, core
+from homeassistant.components.todo import (
+    TodoItem,
+    TodoItemStatus,
+    TodoListEntity,
+    TodoListEntityFeature,
+)
+from homeassistant.core import callback
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import BringDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=90)
+
+
+async def async_setup_entry(
+    hass: core.HomeAssistant,
+    config_entry: config_entries.ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the sensor from a config entry created in the integrations UI."""
+    coordinator = hass.data[DOMAIN][config_entry.entry_id]
+
+    unique_id = config_entry.unique_id
+
+    if TYPE_CHECKING:
+        assert unique_id
+
+    for _list in coordinator.data:
+        async_add_entities(
+            [
+                BringTodoListEntity(
+                    coordinator,
+                    name=f"Bring! {_list['name']}",
+                    unique_id=f"{unique_id} {_list['listUuid']}",
```

Comment:
Why don't we just pass in the list?

Reply:
Because I thought it might be good to know where the list comes from. But on the other hand, the user can do that himself if he needs it.

Reply:
No I mean, can't we pass in the list to the entity instead of iterating over all lists in the entity and setting values outside of the object itself

Reply:
Ahh, I get it. Yes we should be able to do so.